planet/src/main/webapp/places.js

```diff
@@ -101,27 +100,37 @@ function initMap() {
         map.fitBounds(bounds);
         listResults(placeNames);
     });
-
-    // Create the results section for places pagination
-    let service = new google.maps.places.PlacesService(map);
-    let getNextPage = null;
-    let moreButton = document.getElementById('more');
-    moreButton.onclick = function() {
-        moreButton.disabled = true;
-        if (getNextPage) getNextPage();
-    };
-
 }
 
-/** List results of nearby search */
+/** List results of nearby search on page*/
 function listResults(results) {
     const places = document.getElementById('places');
 
-    for (var i = 0; i < results.length; i++) {
-        places.innerText = places.innerText + <br /> + results[i]; 
+    // Displays search results on page
+    for (let i = 0; i < results.length; i++) {
+        // Create materialize card for each result by creating dom element in HTML
+        let div1 = document.createElement("div");
```

Comment:
Why do we need 3 divs? Is there materialize documentation we can link to here?

Reply:
Is there a way to do that? I manually created the materialize card (which required 3 divs)

Reply:
I meant a link in the comment to the documentation that explains what the 3 divs are. Maybe this one? https://materializecss.com/cards.html#horizontal